whylog/constraints/verifier.py

```diff
@@ -182,6 +182,25 @@ def constraints_or(cls, clues_lists, effect, constraints, constraint_manager):
         return causes
 
     @classmethod
+    def constraints_not(cls, clues_lists, effect, constraints, constraint_manager):
+        """
+        provide investigation if there is zero or one constraint,
+        because only in such cases NOT linkage has sense
+        """
+        if len(constraints) > 1:
+            return []
+        if constraints:
+            if clues_lists:
+                return cls.single_constraint_not(
+                    clues_lists, effect, constraints[0], constraint_manager
+                )
+        else:
+            if clues_lists:
+                return [
+                ]  # if all parsers found their matched logs, the NOT requirement isn't satisfied
```

Comment:
reformat this, so that `return []` is preceded by a separate line with a comment